test/lib/utils.test.js

```diff
@@ -76,6 +76,10 @@ describe("lib/utils", () => {
 
   });
 
+  describe("identify", () => {
```

Comment:
Please add the actual test. I know some of the other functions don't have any but I'd like to not get more tech debt added.

Reply:
I was wondering if that was going to be allowed to fly.